pkg/types/installconfig.go

```diff
@@ -34,6 +35,7 @@ var (
 	// hidden-but-supported platform names. This list isn't presented
 	// to the user in the interactive wizard.
 	HiddenPlatformNames = []string{
+		baremetal.Name,
```

Comment:
it looks like you need libvirt features built, so this should be part of the libvirt tag setup or it's own.

Reply:
Can you clarify how this relates to the code you're commenting on? Why do build tags have to do with setting hidden platform names?

Reply:
platforms in hidden platform list don't show up in the TUI.

and my suggestion was rather than dropping baremetal from the TUI.. add it to the list of platforms based on libvirt build tag so that the TUI is accessible to users.

Reply:
I think the idea here was to add an installconfig_baremetal.go like the libvirt one, so a binary built with the baremetal tag would have baremetal included in PlatformNames